remind.py

```diff
@@ -524,9 +539,12 @@ def ics2rem():
 
     parser = ArgumentParser(description='Converter from iCalendar to Remind syntax.')
     parser.add_argument('-l', '--label', help='Label for every Remind entry')
-    parser.add_argument('--tail', help='Tail for every Remind entry')
-    parser.add_argument('--sep', default=" ",
+    parser.add_argument('--tail',        help='Tail for every Remind entry')
```

Comment:
Please use only one space, don't vertical align (this just breaks with new commits!). Also, this should be in the first commit ;).